### PR TITLE
ChefDK 4.1 and 4.2 Release Notes update

### PR DESCRIPTION
**Pull request by mjingle**
Signed-off-by: Mary Jinglewski <mjinglewski@chef.io>

### Description

Updates the ChefDK release notes to include 4.1 and 4.2 details.

### Issues Resolved

https://github.com/chef/chef-web-docs/issues/2027

### Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: chef_master/source/release_notes_chefdk.rst [hunk 1]
```diff
@@ -1,10 +1,158 @@
...
+      .. code-block:: bash
+
+          $ knife ec2 eni list
+          ID                     Status      AZ          Public IP       Private IPs    IPv6 IPs  Subnet ID        VPC ID
+          eni-0123f25ae7805b651  in-use  us-west-2a  63.192.209.236  10.0.0.204               subnet-4ef3b123  vpc-b1bc8d9d
+          eni-2451c913           in-use  us-west-2a  137.150.209.123 10.0.0.245               subnet-4ef3b123  vpc-b1bc8d9d
+
+    * **knife ec2 securitygroup list**
+
+      This command lists all security groups in your environment including the ID, which you need when assigning a newly provisioned instance a group.
```
Comment (IanMadd) [inline chef_master/source/release_notes_chefdk.rst:117]: > assigning a newly provisioned instance a group.

Should it be "... assigning a newly provisioned instance **to** a group."
Or "**in** a group."

File: chef_master/source/release_notes_chefdk.rst [hunk 2]
```diff
@@ -1,10 +1,158 @@
...
+
+    Chef Infra Client has been updated to 15.1 with new and improved resources, improvements to target mode, bootstrap bug fixes, new Ohai detection on VirtualBox hosts, and more. See the `Chef Infra Client 15.1 Release Notes <https://github.com/chef/chef/blob/master/RELEASE_NOTES.md#chef-infra-client-151>`__ for a complete list of new and improved functionality.
+
+  * **Chef InSpec 4.6.9**
+
+    Chef InSpec has been updated from 4.3.2 to 4.6.9 with the following changes:
+
+      * InSpec ``Attributes`` have now been renamed to ``Inputs`` to avoid confusion with Chef Infra attributes.
+      * A new InSpec plugin type of ``Input`` has been added for defining new input types. See the `InSpec Plugins documentation <https://github.com/inspec/inspec/blob/master/docs/dev/plugins.md#implementing-input-plugins>`__ for more information on writing these plugins.
+      * InSpec no longer prints errors to the stdout when passing ``--format json``.
```
Comment (IanMadd) [inline chef_master/source/release_notes_chefdk.rst:53]: Should stdout be in a codeblock? ``stdout``

**Review by IanMadd — COMMENTED** [review]
tenor-103407579
The stdout thing is mostly a quibble, but the other is a little confusing.

**Comment by mjingle** [comment]
This is a duplicate PR of https://github.com/chef/chef-web-docs/pull/2056

**Pull request closed without merging**